scripts/osfstorage/usage_audit.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TEMPDIR=`mktemp -d`
+trap "rm -rf $TEMPDIR" EXIT
+
+export HOME=$TEMPDIR
+cd /opt/apps/osf
+source /opt/data/envs/osf/bin/activate
```

Comment:
Why activate the virtualenv, instead of calling the environment's binary directly? I think the following would do the same thing as lines 8-10:

`/opt/data/envs/osf/bin/python -m scripts.osfstorage.usage_audit send_mail`